### ISSUE TITLE
Mixing Issue with the new_ver

### ISSUE THREAD

**Issue by DevonKerins** [Li-Reactive-Water-Group/HBV-BioRT#40]
**Describe the bug**
It seems there is an issue with the river mixing. When I manually calculate the river concentration (C0Q0 + C1Q1 + C2Q2)/(Q0+Q1+Q2) I get a different river concentration than the model output.

Furthermore, if I change the proportion of Q2 vs. Q1 the concentrations in the river don't seem to reflect the new proportion of ground vs. soil water.

**To Reproduce**
I have uploaded two input files
Case 1. CoalCreek2 has "Results" from HBV where Q2 represents about 20% of river flow
Case 2. CoalCreek3 has "Results" from HBV where Q2 represents about 50% of river flow

**Expected behavior**
I would expect the river concentration output from Case 2 would be closer to the LZ concentration. However, this isn't the case. Case 2 still seems to show about the same relative mixing as Case 1 between the UZ DOC concentration and the LZ DOC concentration.

**Additional context**
![image](https://user-images.githubusercontent.com/70672312/169357038-e163e8c8-ee5f-48ad-b286-4be3d6206731.png)
![image](https://user-images.githubusercontent.com/70672312/169357307-e01bb98b-2f9f-4811-ae1b-fc8f0838a9d5.png)

[CoalCreek2.zip](https://github.com/Li-Reactive-Water-Group/HBV-BioRT/files/8732364/CoalCreek2.zip)
[CoalCreek3.zip](https://github.com/Li-Reactive-Water-Group/HBV-BioRT/files/8732365/CoalCreek3.zip)

**Comment by DevonKerins** [comment]
Sorry, the first figure is wrong - the mixing doesn't seem off for the first case. But rather the second case still holds.

**Comment by KayalvizhiSadayappan** [comment]
Hi, 

I ran the second case (CoalCreek3.zip) file and stream chemistry in results file is correct according to my calculations. I have attached the result files and figure I plotted. Let me know if you are getting different results.

Thanks,
Kayal
[CoalCreek3_results_numexp.txt](https://github.com/Li-Reactive-Water-Group/HBV-BioRT/files/8743651/CoalCreek3_results_numexp.txt)
![image](https://user-images.githubusercontent.com/70716152/169600680-f33e7785-0b55-4f7d-acbd-c62ce75896d4.png)
